### PR TITLE
#22 circle ci integration

### PR DESCRIPTION
**Pull request by costea32**
Integrating tests with Circle CI

### CODE REVIEW

File: junit-tests/src/test/java/org/testmonkeys/webpages/tests/htmlelement/HtmlElementsTest.java [hunk 1]
```diff
@@ -21,10 +25,16 @@
...
+        DesiredCapabilities cap=DesiredCapabilities.chrome();
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--headless");
+        cap.setCapability(ChromeOptions.CAPABILITY,options);
+
+        browser= new Browser(df.initDriver("chrome",cap));
         File file = new File("src/test/resources/WebPages/HtmlElementsPage.html");
         String absolutePath = file.getAbsolutePath();
-        pageFactory=new PageFactory(browser,new PageScanner("org.testmonkeys.webpages"),absolutePath);
+        System.out.println(absolutePath);
```
Comment (monofrei) [inline junit-tests/src/test/java/org/testmonkeys/webpages/tests/htmlelement/HtmlElementsTest.java]: Remove sout or replace with logging

File: koshmar/src/main/java/org/testmonkeys/koshmar/core/browser/DriverFactory.java [hunk 2]
```diff
@@ -2,14 +2,29 @@
...
+import org.openqa.selenium.chrome.ChromeOptions;
 import org.openqa.selenium.remote.DesiredCapabilities;
 
 public class DriverFactory {
 
     public WebDriver initDriver(String type) {
         switch (type.toLowerCase()) {
             case "chrome":
-                return new ChromeDriver(DesiredCapabilities.chrome());
+                DesiredCapabilities cap=DesiredCapabilities.chrome();
```
Comment (monofrei) [inline koshmar/src/main/java/org/testmonkeys/koshmar/core/browser/DriverFactory.java]: Duplicated code in DriverFactory and before scenario

Comment (costea32) [inline koshmar/src/main/java/org/testmonkeys/koshmar/core/browser/DriverFactory.java]: fixed

File: junit-tests/src/test/java/org/testmonkeys/webpages/tests/htmlelement/HtmlElementsTest.java (hunk 1 above)
Comment (costea32) [inline junit-tests/src/test/java/org/testmonkeys/webpages/tests/htmlelement/HtmlElementsTest.java]: fixed

**Pull request merged**